src/Symfony/Component/Validator/Resources/translations/validators.mn.xlf

```diff
@@ -131,20 +131,20 @@
                 <target>Энэ утга зөвхөн тоо байна.</target>
             </trans-unit>
             <trans-unit id="36">
-                <source>This value is not a valid country.</source>
-                <target>Энэ утга үнэн бодит улс биш байна.</target>
-            </trans-unit>
-            <trans-unit id="37">
                 <source>This file is not a valid image.</source>
                 <target>Файл зураг биш байна.</target>
             </trans-unit>
-            <trans-unit id="38">
+            <trans-unit id="37">
                 <source>This is not a valid IP address.</source>
                 <target>IP хаяг зөв биш байна.</target>
             </trans-unit>
-            <trans-unit id="39">
+            <trans-unit id="38">
                 <source>This value is not a valid language.</source>
-                <target>Энэ утга үнэн зөв хэл биш байна .</target>
```

Comment:
this was actually correct

Reply:
Google translated translated this to "This meaning is not accurate". 

Can you (once again) confirm that it is correct?